app/components/member/Edit/form.js

```diff
@@ -12,8 +12,8 @@ import OrganizationMemberUpdateMutation from '../../../mutations/OrganizationMem
 
 import OrganizationMemberRoleConstants from '../../../constants/OrganizationMemberRoleConstants';
 
-class Role extends React.PureComponent {
-  static displayName = "Member.Edit.Role";
+class Form extends React.PureComponent {
+  static displayName = "Member.Edit.Form";
```

Comment:
Yeah totally. I think Relay Modern wants them to be named the right thing, so in this case, the component would be called: `MemberEditForm`. Maybe if we did a sweep and updated names of all the things that'd be a good idea?